BoneLib/BoneLib/HelperMethods.cs

```diff
@@ -53,17 +53,26 @@ public static void SpawnCrate(string barcode, Vector3 position, Quaternion rotat
         /// <param name="crateReference">The crate reference to spawn</param>
         /// <param name="position">The position to spawn the crate at</param>
         /// <param name="rotation">The rotation of the spawned object</param>
-        /// <param name="scale">The scale of the spawned object</param>
         /// <param name="ignorePolicy">Ignore spawn policy or not</param>
         /// <param name="spawnAction">Code to run once the spawnable is placed</param>
-        public static void SpawnCrate(SpawnableCrateReference crateReference, Vector3 position, Quaternion rotation = default, Vector3 scale = default, bool ignorePolicy = false, Action<GameObject> spawnAction = null)
+        public static void SpawnCrate(SpawnableCrateReference crateReference, Vector3 position, Quaternion rotation = default, Vector3 scale = default, bool ignorePolicy = false, Action<GameObject> spawnAction = null, Action<GameObject> despawnAction = null)
         {
             Spawnable spawnable = new Spawnable()
             {
-                crateRef = crateReference,
+                crateRef = crateReference
             };
+
+            var nullableScale = new Il2CppSystem.Nullable<Vector3>
+            {
+                value = scale
+            };
+            var groupId = new Il2CppSystem.Nullable<int>
+            {
+                value = 0
+            };
```

Comment:
DONT DO THIS! **KEEP THE NULLABLE PARAMETERS** and set values, but if the parameters _are null_, just set HasValue to false!!!
This shouldn't do jack shit in the first place, assuming it creates nulalbles with all their values defaulted to 0/false, because HasValue will be set to false anyway, but _still_.